src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
+
+    switch (true) {
+      case type === 'addProperties':
+        Object.assign(state, extraData);
+        break;
+
+      case type === 'clear':
+        for (const key in state) {
+          delete state[key];
+        }
+        break;
+
+      case type === 'removeProperties':
+        for (const key of keysToRemove) {
+          delete state[key];
+        }
+        break;
+    }
+  }
```

Comment:
it would be nice if you added a default value

Reply:
agree, [checklist](https://github.com/mate-academy/js_stateful-object/blob/master/checklist.md) #4